### PR TITLE
added ability to change status when sending message.

### PR DESCRIPTION
**Pull request by danneSvall**
## Types of changes

- [ ] Bug fix
- [X] New feature
- [ ] Removed feature
- [ ] Code style update (formatting etc.)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Content/Data

## Does this PR introduce a breaking change?
- [ ] Yes (I have stepped the version number accordingly)
- [X] No
      
## Checklist:

- [X] My code follows the code style of this project.
- [ ] I have updated the documentation accordingly (if applicable).
- [ ] I have added/updated tests to cover my changes (if applicable).

Jira: https://jira.sundsvall.se/browse/DRAKEN-683

### CODE REVIEW

File: frontend/src/supportmanagement/components/support-message-form/support-message-form.component.tsx [hunk 1]
```diff
@@ -295,6 +302,13 @@ export const SupportMessageForm: React.FC<{
         setTimeout(() => {
           props.update();
         }, 500);
+
+        if (typeOfMessage === 'infoCompletion') {
+          setSupportErrandStatus(supportErrand.id, municipalityId, Status.PENDING);
```
Comment (hensankom) [inline frontend/src/supportmanagement/components/support-message-form/support-message-form.component.tsx:307]: Undrar om man ska sätta "await" på setSupportErrandStatus-anropen här så att catchen hinner snappa upp eventuella fel som uppstår. 

Då får man även sätta "async" i then(... på rad 281

Comment (danneSvall) [reply frontend/src/supportmanagement/components/support-message-form/support-message-form.component.tsx:307]: jaa de kanske man borde göra! fixar de!

Comment (danneSvall) [reply frontend/src/supportmanagement/components/support-message-form/support-message-form.component.tsx:307]: fast då verkar den inte vilja byta fas alls, plus att gränssnittet verkar inte vilja renderar om.

Comment (hensankom) [reply frontend/src/supportmanagement/components/support-message-form/support-message-form.component.tsx:307]: Hm!

**Review by hensankom — APPROVED** [review]

**Pull request merged**